changes-set.txt

```diff
@@ -26,6 +26,7 @@ make a github issue.)
 
 DONE:
 Date      Old       New         Notes
+24-Aug-23 cdaassen  djuassen    Changes from +c notation to |_|
```

Comment:
What about the other theorems, e.g. cdacomen - djucomen etc.?

Reply:
I don't know, would you put them in the "PROPOSED" section?

The ones that are listed in the pull request now are the ones which are removed, not the ones marked as discouraged.

Reply:
OK, if you add them when they are removed.